modules/kubefed-deregistering-clusters.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+// * kubefed/kubefed-resource-federation.adoc
+
+[id="kubefed-deregistering-clusters_{context}"]
+= Deregistering (unjoining) clusters
+
+The `unjoin` operation reads information about how to disconnect the registering cluster from the cluster hosting federation from your local `kubeconfig` file.
+
+You can use the `kubefedctl unjoin` command to deregister clusters from {KubeFedProductShortName} resource federation.
+
+[NOTE]
+====
+This command deletes the KubeFedCluster resource for the deregistering cluster.
+====
+
+.Procedure
+
+. Use the `kubefedctl unjoin` command with the required arguments as shown in the example.
++
+----
+$ kubefedctl unjoin <cluster-name> --cluster-context <cluster-context> --host-cluster-context <cluster-context> --v=2
+----
++
+[NOTE]
+====
+`cluster-context` will default to using the registering cluster name if not specified.
+====
```

Comment:
Deregistering command "kubefedctl unjoin"  has the "--kubefed-namespace" parameters too.